iam.tf

```diff
@@ -6,7 +6,7 @@ resource "aws_iam_instance_profile" "default" {
 
 resource "aws_iam_role" "default" {
   count = (module.this.enabled && local.instance_profile_count == 0) ? 0 : 1
-  name  = module.this.id
+  name  = "${module.this.id}-${data.aws_region}"
```

Comment:
I don't believe we'd accept this change as 1) I don't believe this would be a valid reference (i.e. it would fail) and 2) the develop can add the region (or region identifier) to their name via the context variables that make up `module.this.id`.

Can you please remove this and the rename from `default` => `current` before we potentially move this PR forward?

Reply:
Didn't mean for this to be included. This PR was based on our master & not the volume-type specific branch.

I've based a PR on that branch & "started over" here: https://github.com/cloudposse/terraform-aws-ec2-bastion-server/pull/77